scripts/setup.py

```diff
@@ -23,20 +23,23 @@
 import sys
 import stat
 import time
-from concurrent.futures import as_completed
 from functools import partialmethod
 from itertools import chain
 from pathlib import Path
 
-from addict import Dict as NSDict
-
-import util
 from util import (
```

Comment:
For sake of code clarity I would recommend to not use `from X import Y`, but `import X` instead.
`util.run(...)` is better than `run(...)`.

I don't ask you to re-do all imports here , but pls don't remove `import util`.

Reply:
Ack, updated.